src/lib/components/Buttons.svelte

```diff
@@ -0,0 +1,7 @@
+<script>
+    import Button1 from '$lib/components/Button1.svelte';
+    import Button2 from '$lib/components/Button2.svelte';
```

Comment:
We could use a more descriptive file name and import name for this. Perhaps `CsiBlueButton.svelte` and `BlackButton.svelte` would be nice?